lib/spack/spack/test/install.py

```diff
@@ -38,6 +39,8 @@ def find_nothing(*args):
         with pytest.raises(spack.repo.UnknownPackageError):
             spec.package
 
+        module_path = spack.modules.common.get_module('tcl', spec, True)
```

Comment:
Can you factor this out into a separate test? This should be a test of the module system, not a test of the install method.

Reply:
moved to modules/common.py